src/Fhir.Anonymizer.Shared.FunctionalTests/Fhir.Anonymizer.Shared.FunctionalTests.projitems

```diff
@@ -0,0 +1,119 @@
+﻿<?xml version="1.0" encoding="utf-8"?>
+<Project xmlns="http://schemas.microsoft.com/developer/msbuild/2003">
+  <PropertyGroup>
+    <MSBuildAllProjects>$(MSBuildAllProjects);$(MSBuildThisFileFullPath)</MSBuildAllProjects>
+    <HasSharedItems>true</HasSharedItems>
+    <SharedGUID>19e3e7a2-17a6-4708-b9f2-04a2c9968c83</SharedGUID>
+  </PropertyGroup>
+  <PropertyGroup Label="Configuration">
+    <Import_RootNamespace>Fhir.Anonymizer.Shared.FunctionalTests</Import_RootNamespace>
+  </PropertyGroup>
+  <ItemGroup>
+    <None Include="$(MSBuildThisFileDirectory)Configurations\r4-configuration-sample.json">
```

Comment:
Are these config files used in both versions? If not, we should keep them in version specific projects.